src/surabhi/BankDetails.java

```diff
@@ -0,0 +1,74 @@
+package surabhi;
+
+/*Create a Banking System which has the following functionality.
+
+Create two object to perform below scenario.
+1) Method to debit amount
+2) Method to credit amount
+3) Method to printBalance [it will print current balance]
+4) individualTransactionSummary() method should show how many times individually debit, credit & printBalance method called .
+Hint : Create 3 non static instance variable.
+5) allTransactionSummary() method should show by both users total how many times debit, credit & printBalance method get called.
+Hint : Create 3 static variables will be required.
+*/
+
+public class BankDetails {
+	String Customername;
+	int currentBal=10000;
```

Comment:
Not require to give 0 value to variable, if you are not initialised any variable with any value then it will initialised with it's default value. and this balance should not hard coded.

Reply:
Noted.